upcoming-release-notes/1988.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [carkom]
+---
+
+Custom Reports optimization.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏people reading the changelog won't understand what you've actually done in this change. It would be nice if you could write out what this PR is improving. For example: `Data loading performance improvements for custom reports`. The current "optimizations" is very vague.